.github/workflows/test.yml

```diff
@@ -365,6 +373,7 @@ jobs:
         with:
           prefix-key: "v1-${{ runner.os }}-rust-cache"
           shared-key: "cli-e2e"
+          save-if: ${{ !startsWith(github.ref, 'refs/heads/gh-readonly-queue') }}
```

Comment:
what does this mean?

Reply:
it means don't write to cache if we're in the merge queue (since no other branches can reuse it)